run.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+if [ "$#" -ne 1 ]; then
+  echo "Usage: $0 [start|watch|serve|clean|all] app=<app_name> port=8001"
```

Comment:
the `app=<app_name> port=8001` part wouldn't pass the parameters through to make, would it?